index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>
+    Document
+  </title>
+</head>
+
+<body>
+  <div class=border>
```

Comment:
All attributes, including class names, should be surrounded in quotes, like so:
```suggestion
  <div class="border">
```

Also, consider swapping out this `div` with a `main` element?